rust-runtime/aws-smithy-compression/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aws-smithy-compression"
-version = "0.0.2"
+version = "0.0.3"
```

Comment:
The package versions were bumped [per check-rust-runtime](https://github.com/smithy-lang/smithy-rs/actions/runs/12897911115/job/35970402689) for crates that were not already modified as a result of the hyper1 branch changes.